setup_map_sra_to_ontology/build_bk_tree.py

```diff
@@ -30,7 +30,8 @@ def main():
                 string_identifiers.add(syn.syn_str)
 
     print ("Building the BK-Tree...")
-    bk_tree = BKTree(string_metrics.bag_dist_multiset, string_identifiers)
+    #bk_tree = BKTree(string_metrics.bag_dist_multiset, string_identifiers)
```

Comment:
We can remove old commented out code

Reply:
I would like to keep it as there are at least two ways to call BKTree. And one of them is commented and maybe later un-commented.

Reply:
If so, I would add a more descriptive comment about why this code is left commented out.